build.gradle

```diff
@@ -30,7 +30,8 @@ buildscript {
         'androidx_test_runner'      : '1.4.0',
         'espresso'                  : '3.4.0',
         'uiautomator'               : '2.2.0',
-        'ktlint'                    : '0.47.1'
+        'ktlint'                    : '0.47.1',
+        'cash_app_sdk'              : '1.0.2'
```

Comment:
nit: we call it (legal's call), "Cash App PayKit SDK". Its a mouthful, but perhaps we should use that longer naming for consistency

Reply:
Addressed in [bfc10e8](https://github.com/afterpay/sdk-android/pull/184/commits/bfc10e8d3f8d5052f9367d0e85fe1326c0928f61)